bin/dropbox_setup.py

```diff
@@ -28,11 +28,11 @@ def do_exit(self, cmd):
 
 	def do_list(self, cmd):
 		"""list: lists the dropbox usernames."""
-		print
+		self.stdout.write("\n")
```

Comment:
`print("")`

(or simply `print()` with `from __future__ import print_function`)

Reply:
I tend to use `self.stdout` instead of `print` in `cmd.Cmd` subclasses for code reusability. `self.stdout` is an attribute of `cmd.Cmd` and may not be `sys.stderr`.

Reply:
Ah, I misread that - I thought it said `sys.stdout`. I didn't know that `self.stdout` existed here. Thanks for the explanation!